pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper_test.go

```diff
@@ -39,9 +39,54 @@ func TestGetNumPrPages(t *testing.T) {
 			expectedErr:   nil,
 			expectedPages: 2,
 		},
+		{
+			desc:          "error on open pr count",
+			client:        &mockClient{openPrCount: 10, err: true, errString: "this is an error"},
```

Comment:
Do these `errString` values get surfaced? If so, do we need to set these to actual values?